pkg/cloud/services/securitygroup/securitygroups.go

```diff
@@ -799,3 +785,27 @@ func ingressRulesFromSDKType(v *ec2.IpPermission) (res infrav1.IngressRules) {
 
 	return res
 }
+
+func (s *Service) getDefaultIngressRulesForControlPlaneLB() infrav1.IngressRules {
+	if s.scope.VPC().IsIPv6Enabled() {
+		return infrav1.IngressRules{
+			{
+				Description:    "Kubernetes API IPv6",
+				Protocol:       infrav1.SecurityGroupProtocolTCP,
+				FromPort:       int64(s.scope.APIServerPort()),
+				ToPort:         int64(s.scope.APIServerPort()),
+				IPv6CidrBlocks: []string{services.AnyIPv6CidrBlock},
+			},
+		}
+	} else {
```

Comment:
Don't need the else, you are already returning in the above section. :)

Reply:
I mean, drop the else, and just return the `IngressRule`.

Reply:
Done. Please take a look.